### PR TITLE
Feat/add analytics tracking

### PR DESCRIPTION
**Pull request by AnisLahouar**
Add analytics to:
jukebox
Doge NPC
Cloud Region
Bar Region
event/slider towers

### CODE REVIEW

File: src/aiNpc/Stats/Segment.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    const doc: any = {
+      sceneId: sceneId,
+      realm: realm,
+      spanId: instance,
+      elementType: elementType,
+      elementId: elementId,
+      event: event,
+      selection: selection,
+      durationTime: inSeconds ? durationTime * 0.001 : durationTime,
```
Comment (wacaine) [inline src/aiNpc/Stats/Segment.ts:45]: inSeconds is a flag, should be all caps

File: src/aiNpc/Stats/Segment.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+      playTime: Date.now(),
```
Comment (wacaine) [inline src/aiNpc/Stats/Segment.ts:47]: I think playTime == from start till now,  not the current moment.  that is tracked by  timestamp: new Date().toJSON(),

https://github.com/decentraland/dcl-onboarding/blob/main/src/genesis/stats/segment.ts#L14

**Review by wacaine — CHANGES_REQUESTED** [review]
see coments

File: src/aiNpc/Stats/genesis.data.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+//Global access to the game data
+class GenesisData {
+
+    readonly startPlayTime: number = Date.now()
+
+    private static instanceRef: GenesisData;
+
+    bdeploy: boolean = false;
```
Comment (wacaine) [inline src/aiNpc/Stats/genesis.data.ts:11]: @AnisLahouar  spelling mistake?

Comment (AnisLahouar) [reply src/aiNpc/Stats/genesis.data.ts:11]: in all honesty, idk, i just used the same file from onboarding and removed the extra elements, that one slipped by me

**Pull request merged**